server-data/resources/[framework]/tcrp-wagons/shared/config.lua

```diff
@@ -0,0 +1,271 @@
+            ["names"] = "Coach 1",
+        },
+        [5] = {
+            ["coords"] = vector3(2532.56, -1358.31, 46.55),
+            ["heading"] = 2.06,
+            ["model"] = "coach4",
+            ["price"] = 400,
+            ["names"] = "Small Coach 1",
+        },
+    },
+
+    ["Annesburg"] = {
+        [1] = {
+            ["coords"] = vector3(2982.09, 1419.61, 44.75),
+            ["heading"] = 123.95,
+            ["model"] = "cart05",
+            ["price"] = 100,
+            ["names"] = "Small Barrel Cart",
+        },
+        [2] = {
+            ["coords"] = vector3(2976.81, 1429.36, 44.71),
+            ["heading"] = 224.35,
+            ["model"] = "CART06",
+            ["price"] = 100,
+            ["names"] = "Small Cart 6",
+        },
+        [3] = {
+            ["coords"] = vector3(2968.88, 1430.81, 44.72),
+            ["heading"] = 219.07,
+            ["model"] = "wagon02x",
+            ["price"] = 300,
+            ["names"] = "Large Wagon 2",
+        },
+    },
+
+    ["Blackwater"] = {
+        [1] = {
+            ["coords"] = vector3(-957.9, -1325.25, 51.94),
+            ["heading"] = 93.13,
+            ["model"] = "wagon03x",
+            ["price"] = 350,
+            ["names"] = "Large Seated Wagon",
+        },
+        [2] = {
+            ["coords"] = vector3(-981.74, -1318.25, 51.9),
+            ["heading"] = 270.78,
+            ["model"] = "wagon06x",
+            ["price"] = 250,
+            ["names"] = "Small Cargo Wagon",
+        },
+        [3] = {
+            ["coords"] = vector3(-982.08, -1312.23, 51.75),
+            ["heading"] = 270.78,
+            ["model"] = "cart07",
+            ["price"] = 120,
+            ["names"] = "Small Cart 7",
+        },
+        [4] = {
+            ["coords"] = vector3(-981.5, -1305.78, 52.21),
+            ["heading"] = 270.78,
+            ["model"] = "coach5",
+            ["price"] = 270,
+            ["names"] = "Coach 5",
+
+        },
+    },
+
+    ["Tumbleweed"] = {
+        [1] = {
+            ["coords"] = vector3(-5531.65, -3026.41, -1.53),
+            ["heading"] = 15.0,
+            ["model"] = "cart04",
+            ["price"] = 100,
+            ["names"] = "Small Cart 4",
+        },
+        [2] = {
+            ["coords"] = vector3(-5527.95, -3020.56, -1.67),
+            ["heading"] = 288.48,
+            ["model"] = "chuckwagon000x",
+            ["price"] = 250,
+            ["names"] = "Chuck Wagon",
+        },
+        [3] = {
+            ["coords"] = vector3(-5524.13, -3033.15, -2.25),
+            ["heading"] = 114.05,
+            ["model"] = "wagontraveller01x",
+            ["price"] = 150,
+            ["names"] = "Travellers Wagon",
+        },
+        [4] = {
+
+            ["coords"] = vector3(-5521.23, -3021.03, -2.24),
+            ["heading"] = 199.99,
+            ["model"] = "cart03",
+            ["price"] = 100,
+            ["names"] = "Small Cart 3",
+
+        },
+    }
+}
+  
+    
+--[[
```

Comment:
whats this for?

Reply:
I honestly don't know, looks all commented out.

Reply:
You can get rid of this of move it over to discord it seems like a devs list to keep track of what vehicles are currently used in their server